nupic/frameworks/opf/clamodel.py

```diff
@@ -671,6 +666,12 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
         "No predicted field was enabled! Did you call enableInference()?"
       )
     self._predictedFieldName = predictedFieldName
+    # check if predicted field was correctly specified
+    if not self._predictedFieldName in self._input:
```

Comment:
If an instance attribute is passed as a function parameter then you should always use the passed in parameter because it is much easier to test the function and would be confusing if you pass a parameter in but it doesn't get used.